api/lightwallet_rest.md

```diff
@@ -309,11 +318,12 @@ was actually spent.
 
 **Response** object
 
-|    Field   |        Type       |                Description              |
-|------------|-------------------|-----------------------------------------|
-| per_kb_fee | uint64            | Estimated network fee                   |
-| amount     | uint64-string     | The total value in outputs              |
-| outputs    | array of output's | Outputs possibly available for spending |
+|    Field     |        Type       |                Description              |
+|--------------|-------------------|-----------------------------------------|
+| per_byte_fee | uint64-string     | Estimated network fee                   |
```

Comment:
This field is currently a `uint64` for the same reason, this was not a typo.

Reply:
Yeah, @moneroexamples requested all XMR amount fields to be `uint64-string` for consistency. It seemed reasonable enough, assuming the costs of parsing the string are negligible. Is that not the case?